src/sonic-config-engine/sonic_yang_cfg_generator.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import print_function
+
+import json
+import sys
+
+
+import sonic_yang
+# TODO: Remove this once we no longer support Python 2
+if sys.version_info.major == 3:
+    UNICODE_TYPE = str
+else:
+    UNICODE_TYPE = unicode
+"""sonic_yang_cfg_generator
```

Comment:
>sonic_yang_cfg_generator [](http://example.com/codeflow?start=3&length=24)

This seems for the whole file, could you move it to top?

Reply:
Or for the class? Then move it near the class.

Reply:
Fixed in the latest commit